src/main/java/de/caritas/cob/userservice/api/admin/facade/ConsultantAdminFacade.java

```diff
@@ -60,13 +64,34 @@ public ConsultantAdminResponseDTO findConsultant(String consultantId) {
    */
   public ConsultantSearchResultDTO findFilteredConsultants(Integer page, Integer perPage,
       ConsultantFilter consultantFilter, Sort sort) {
+    validateSortInputField(sort);
     var filteredConsultants = this.consultantAdminFilterService
         .findFilteredConsultants(page, perPage, consultantFilter, sort);
     retriveAndMergeAgenciesToConsultants(filteredConsultants);
 
     return filteredConsultants;
   }
 
+  private void validateSortInputField(Sort sort) {
+    var containsNoValidField = Stream.of(FieldEnum.values())
+        .noneMatch(providedSortFieldIgnoringCase(sort));
+
+    if (containsNoValidField) {
+      sort.setField(FieldEnum.LASTNAME);
+      sort.setOrder(OrderEnum.ASC);
+    }
+  }
+
+  private Predicate<FieldEnum> providedSortFieldIgnoringCase(Sort sort) {
+    return field -> {
+      if (nonNull(sort.getField())) {
```

Comment:
I think carinet is using the endpoint without any sort parameter. Could you also cover the case if sort is null?

Reply:
done.